client/landing/jetpack-cloud/sections/backups/activity/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import BackupActivityLog from 'landing/jetpack-cloud/sections/backups/components/activity-log';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+const BackupActivityPage: FunctionComponent = () => {
```

Comment:
Name suggestion `BackupActivityLogPage` to use the same naming as the design and to differentiate it from the `BackupActivityLog` component

![image](https://user-images.githubusercontent.com/9832440/78248386-7440f100-74e4-11ea-9a03-40c2a38e44cb.png)